src/test/cli-test.ts

```diff
@@ -26,12 +29,43 @@ describe('cli', () => {
       error: s => (error += s),
       info: s => (info += s),
     }
-    await cli.run(() => ai.fromSequence(), report)
+    await cli.run(() => ai.fromSequence<IErrorBase>(), report)
     assert.strictEqual(process.exitCode, 0)
     assert.strictEqual(error, '')
     assert.strictEqual(info, 'errors: 0')
   })
-  it('with errors', async () => {
+  it('Error errors', async () => {
```

Comment:
It's odd for the name. Can we keep origin scenario name `with errors` and use `with warnings` for the new test.